Sources/PointFree/RunnerTasks/ValidateEnterpriseEmails.swift

```diff
@@ -0,0 +1,92 @@
+import Either
+import Mailgun
+import Models
+import PointFreePrelude
+import Prelude
+
+public func validateEnterpriseEmails() -> EitherIO<Error, Prelude.Unit> {
+
+  return Current.database.fetchEnterpriseEmails()
+    .flatMap(validate(enterpriseEmails:))
+    .flatMap(sendValidationSummaryEmail(results:))
```

Comment:
these lines fetch the enterprise emails, validate them, and send a summary email (currently fatalErrors, but will fix soon)